app/routes/events.js

```diff
@@ -0,0 +1,12 @@
+const router = require('express').Router();
+const bodyParser = require('body-parser');
+
+const { searchEvents, createEvent, deleteEvent } = require('../controllers/events');
+
+module.exports = (app, conf) => {
+  router.get('/events', searchEvents);
+  router.post('/events/create', bodyParser.json({ limit: '50mb' }), createEvent);
```

Comment:
why we use here `bodyParser.json({ limit: '50mb' })`?

Reply:
I update this code. Now bodyParser [use](https://github.com/inst-event-emitter/eventemitter-backend/pull/10/commits/1f13afa3ee003475496a914607bd85b5320b7020#diff-de77b05ed669485f2114729c25dfe2b3R8) default body limit